mne/chpi.py

```diff
@@ -264,24 +270,52 @@ def _fit_chpi_pos(coil_dev_rrs, coil_head_rrs, x0):
     return x, 1. - result / denom
 
 
+def _fit_dev_head_trans(dev_pnts, head_pnts):
+    """Compute Device to Head transform allowing for permutiatons of points."""
+    import itertools
+    id_quat = np.concatenate([rot_to_quat(np.eye(3)), [0.0, 0.0, 0.0]])
+    best_order = None
+    best_g = -999
+    best_quat = id_quat
+    for i in itertools.permutations(np.arange(len(head_pnts))):
+        head_pnts_tmp = head_pnts[np.array(i)]
```

Comment:
yeah i is a tuple, so indexing doesn't work. I could probably do list if that is preferable?